package.json

```diff
@@ -22,7 +22,5 @@
     "start": "node node_modules/react-native/local-cli/cli.js start"
   },
   "dependencies": {
-    "react": "^16.0.0-alpha.12",
-    "react-native": "^0.45.1"
```

Comment:
This is no longer necessary, when you init a new app these dependencies already exist.